src/components/ReduxFormCheckbox/index.jsx

```diff
@@ -12,10 +12,7 @@ const ReduxFormCheckbox = (props) => {
   } = props;
 
   return (
-    <ValidationFormGroup
-      for={id}
-      helpText={helptext}
-    >
+    <Form.Group controlId={id}>
       <Form.Check
```

Comment:
Should this be `Form.Checkbox`? I don't believe `Form.Check` is a newer Paragon form component; I think it's a subcomponent from react-bootstrap but is otherwise undocumented. Probably shouldn't be accessible via Paragon considering we have `Form.Checkbox` :)